src/rtcp/bye.ts

```diff
@@ -0,0 +1,229 @@
+import { isRtcp, RtcpPacket, RtcpPacketType } from './';
+
+/**
+        0                   1                   2                   3
+        0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+header |V=2|P|    SC   |   PT=BYE=203  |             length            |
+       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+       |                           SSRC/CSRC                           |
+       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+       :                              ...                              :
+       +=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+(opt)  |     length    |               reason for leaving            ...
+       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ */
+
+/** @ignore */
+const SSRC_LENGTH = 4; // SSRC/CSRC length.
+
+/** @ignore */
+const FIXED_HEADER_LENGTH = 4; // Common RTCP header length.
+
+/**
+ * ```ts
+ * import { ByePacket } from 'rtp.js';
+ * ```
+ *
+ * Representation of a RTCP Bye packet.
+ */
+export class ByePacket extends RtcpPacket
+{
+	// Packet Type.
+	static packetType = RtcpPacketType.BYE;
+
+	// SSRC/CSRC array.
+	private ssrcs: number[] = [];
+
+	// Termination season.
+	private reason: string = '';
```

Comment:
This seems to be an optional field so better make it optional (so it defaults to `undefined`):

```suggestion
	private reason?: string;
```